roles/redpanda_broker/defaults/main.yml

```diff
@@ -40,6 +40,15 @@ airgap_copy_dest: "/tmp"
 
 # console
 install_console: false
+console_http_listen_port: 80
+console_https_listen_port: 443
+
+# sasl
+sasl_bootstrap_admin_user: "admin"
+sasl_bootstrap_admin_pw: "password"
+console_sasl_user: "admin"
+console_sasl_pw: "password"
```

Comment:
do we abbreviate pw usually?

Reply:
No, but it felt really long 🤷‍♀️  I can change it